src/CarwowTheme/Filters.elm

```diff
@@ -164,15 +204,14 @@ standardFilterView label selectedIcon items =
                 firstSelectedFilter
     in
         if itemsCount > 1 then
-            filterView label selectedFiltersLabel selectedIcon content
+            filterView label selectedFiltersLabel selectedIcon content alignment
```

Comment:
What are valid values of alignment? Seems like probably not just any-String. A union type perhaps?